examples/rnn.fsx

```diff
@@ -50,13 +50,14 @@ let modelFileName = "rnn_language_model.params"
 if File.Exists(modelFileName) then 
     printfn "Resuming training from existing model params found: %A" modelFileName
     languageModel.state <- dsharp.load(modelFileName)
+    languageModel.move(Device.Default)
```

Comment:
Shouldn't this be the default device?

Reply:
Yes, I think `dsharp.load` loading into the default device makes sense, and that's what seems to be the goal of PR #430.

Pytorch is a little different, it would torch.load into whatever device the tensor was saved from:  "They are first deserialized on the CPU and are then moved to the device they were saved from" ([link](https://pytorch.org/docs/stable/generated/torch.load.html#torch.load)).